apps/fluent-tester/src/FluentTester/TestComponents/ButtonExperimental/ButtonIconTestSection.tsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import { Button } from '@fluentui-react-native/experimental-button';
+import * as React from 'react';
+import { Platform, View } from 'react-native';
+import { commonTestStyles, stackStyle } from '../Common/styles';
+import { SvgIconProps } from '@fluentui-react-native/icon';
+import TestSvg from './test.svg';
+
+export const ButtonIconTest: React.FunctionComponent = () => {
+  const fontBuiltInProps = {
+    fontFamily: 'Arial',
+    codepoint: 0x2663,
+    fontSize: 32,
+  };
+
+  const testImage = require('./icon_24x24.png');
```

Comment:
if we changed the require(icon.png) to an import, could we get rid of the aslant disabling comment?

Reply:
If it's the only require, can we move the disabling comment to just above this line? I think that will ensure it only applies to the one line instead of the whole file, in case more require() calls re added in the future.

Reply:
I could move it, but I also copied it from the ButtonIconTest.tsx for the non-experimental button where it's put as the first line of the file. I just kept it as the first line to be consistent.